README.md

```diff
@@ -19,7 +19,7 @@ and other issues with some mods.
 
 ### Manual installation (recommended)
 
-You will need Fabric Loader 0.10.x or newer installed in your game in order to load Sodium. If you haven't installed
+You will need Fabric Loader 0.12.2 or newer installed in your game in order to load Sodium. If you haven't installed
```

Comment:
Why 0.12.2? 
Do not work in old versions?

Reply:
0.10.x is for older versions of the game. 1.12.2 is the one for 1.18.1